website/docs/tutorials/working-with-files.md

```diff
@@ -4,8 +4,127 @@ title: Working with Files
 sidebar_label: Working with Files
 ---
 
+Even if the core Solid data model is **structured** data (see [Working with data](./working-with-data)), a Solid Pod
+can also act as a regular general-purpose data store. Besides your profile document, your friend list and the likes, your
+Pod can also store your photos, PDFs and any other type of file. Note that the [access restrictions](./managing-access)
+apply to files the same way they apply to any other resource.
```

Comment:
```suggestion
apply to files the same way they apply to any other [Resource](../glossary#resource).
```

(I also typically have the habit of capitalising all RDF/Solid-specific terms, so that readers can know whether they're supposed to know the term because they speak English, or if it's an actual concept that they might want to know more about.)